`Web docs` - Key value inputs (#2941)
hashicorp/design-system: website/docs/components/form/key-value-inputs/index.md

---
title: Key Value Inputs
description: >-

>   Rows of inputs that can be added and deleted by end users.

Reviewer: ```suggestion
  Rows of related inputs that can be individually added & deleted.
```

Reviewer: Since it's not required to have multiple rows it could make sense to instead say:

"A row of related inputs that can be individually added & deleted."

> caption: Rows of inputs that can be added and deleted by end users.

Reviewer: ```suggestion
caption: Rows of related inputs that can be individually added & deleted.
```

links:
  figma: >-

>     https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?node-id=67397-98270&t=w8xQlWxzH7bwXLe2-1

Reviewer 2: I think this link is currently broken, you might need to update it when you actually merge the component branch.

  github: >-
    https://github.com/hashicorp/design-system/tree/main/packages/components/src/components/hds/form/key-value-inputs
related:
  - components/form/text-input
  - components/form/masked-input
  - components/form/select

>   - components/form/textarea

Reviewer 3: Probably better to not link them at all. There are already enough related components. Don't want to add too many. Plus the patterns are usually linked to in the guidelines if relevant.

previewImage: assets/illustrations/components/form/key-value-inputs.jpg
navigation:
  keywords:
    - key
    - value
    - pair
    - input
    - form
---

<section data-tab="Guidelines">
  @include "partials/guidelines/guidelines.md"
</section>

<section data-tab="Code">
  @include "partials/code/how-to-use.md"
  @include "partials/code/component-api.md"
</section>

<section data-tab="Specifications">
  @include "partials/specifications/anatomy.md"
</section>

<section data-tab="Accessibility">
  @include "partials/accessibility/accessibility.md"
</section>

> <section data-tab="Version history">

Reviewer 4: That's why we added it to the generator blueprint.
